src/libs/OptionsListUtils.js

```diff
@@ -436,6 +451,15 @@ function getLastMessageTextForReport(report) {
         lastMessageTextFromReport = lodashGet(lastReportAction, 'message[0].text', '');
     } else if (ReportActionUtils.isCreatedTaskReportAction(lastReportAction)) {
         lastMessageTextFromReport = TaskUtils.getTaskCreatedMessage(lastReportAction);
+    } else if (ReportUtils.isArchivedRoom(report)) {
+        const archiveReason =
+            (lastReportActions[report.reportID] && lastReportActions[report.reportID].originalMessage && lastReportActions[report.reportID].originalMessage.reason) ||
```

Comment:
Should we also make sure the .reason is one of the 3 options?

Reply:
You mean one of these 3: `CONST.REPORT.ARCHIVE_REASON.ACCOUNT_CLOSED`, `CONST.REPORT.ARCHIVE_REASON.REMOVED_FROM_POLICY`, `CONST.REPORT.ARCHIVE_REASON.POLICY_DELETED`? Like it was checked in [SidebarUtils](https://github.com/Expensify/App/blob/c0608977d90835b34382325732e755733c034240/src/libs/SidebarUtils.ts#L378-L391)? Can we get another reason (apart from these [ones](https://github.com/Expensify/App/blob/c0608977d90835b34382325732e755733c034240/src/languages/en.ts#L507-L517))? Actually, now I see that we were not showing the `MERGED` reason there - we would also need to pass the `oldDisplayName` if so. Shou,d it be displayed or should we display `DEFAULT` instead

Reply:
Lets focus on keeping the current behaviour in place. I think we can skip adding the merged consideration for now